deployers/whiskclient.go

```diff
@@ -28,166 +28,166 @@ import (
 	"github.com/apache/incubator-openwhisk-client-go/whisk"
 	"github.com/apache/incubator-openwhisk-wskdeploy/parsers"
 	"github.com/apache/incubator-openwhisk-wskdeploy/utils"
-    "path"
-    "errors"
+	"path"
+	"errors"
 )
 
 const (
-    COMMANDLINE = "wskdeploy command line"
-    DEFAULTVALUE = "default value"
-    WSKPROPS = ".wskprops"
-    WHISKPROPERTY = "whisk.properties"
-    INTERINPUT = "interactve input"
+	COMMANDLINE = "wskdeploy command line"
```

Comment:
David, I was wondering what IDE you used to write go code.
As you see, your PR contains a lot of changes regarding indentations like this, which do not bring any changes. I am using INTELLIJ, and there should be 4 spaces for each indentation. I suspect you use 8 or 7 spaces for each indentation. If so, would you please change it to 4?
Let's make our dev environment consistent, so that we only submit PRs with meaningful changes.